tests/tasks/rm/rm.cue

```diff
@@ -0,0 +1,90 @@
+package main
+
+import (
+	"dagger.io/dagger"
+	"dagger.io/dagger/core"
+)
+
+dagger.#Plan & {
+	actions: {
+		pull: core.#Pull & {
+			source: "alpine:3.15.0@sha256:e7d88de73db3d3fd9b2d63aa7f447a10fd0220b7cbf39803c803f2af9ba256b3"
+		}
+
+		test: {
+			rmFile: {
+				// Write a new file
+				write: core.#WriteFile & {
+					input:    pull.output
+					path:     "/test.txt"
+					contents: "1,2,3"
+				}
+
+				// Remove file
+				rm: core.#Rm & {
+					input: write.output
+					path:  write.path
+				}
+
+				verify: core.#Exec & {
+					input: rm.output
+					args: ["/bin/sh", "-c", """
+						test ! -e /test.txt
+						"""]
+				}
+			}
+
+			rmWildcard: {
+				// Write directory
+				write: core.#Exec & {
+					input: pull.output
+					args: ["/bin/sh", "-c", """
+						touch /foo.txt
+						touch /bar.txt
+						touch /data.json
+						"""]
+				}
+
+				// Remove all .txt file
+				rm: core.#Rm & {
+					input: write.output
+					path:  "/*.txt"
+				}
+
+				verify: core.#Exec & {
+					input: rm.output
+					args: ["/bin/sh", "-c", """
```

Comment:
```suggestion
					args: ["/bin/sh", "-e", "-c", """
```

Missed this from my last comment. Providing `-e` is important here, otherwise the `sh` command will only fail if the final command fails. I.e.:
```
sipsma@dagger_dev:~/repo/github.com/sipsma/dagger$ sh -c 'false; true'
sipsma@dagger_dev:~/repo/github.com/sipsma/dagger$ echo $?
0
sipsma@dagger_dev:~/repo/github.com/sipsma/dagger$ sh -e -c 'false; true'
sipsma@dagger_dev:~/repo/github.com/sipsma/dagger$ echo $?
1
```

Reply:
Nice catch! Updated in https://github.com/dagger/dagger/pull/2422/commits/09bdec156b1a92981e694f1acbe9e00f99d6b99f